live/README.md

```diff
@@ -44,6 +44,19 @@ access services that were started.
 
 ## ISO Creation
 
+We've tested using Amazon EC2 t2.xlarge (for 16GB RAM) running Ubuntu Server 18.04 with 60GB disk storage.
+To setup and install dependencies, run:
+```bash
+git clone https://github.com/posm/posm-build.git
+cd posm-build
+git submodule update --init 
+cd subiquity && make install_deps
+cd ../
+sudo apt install make lxd pigz snapcraft p7zip-full xorriso isolinux
+lxd init  # accepting all defaults
+```
+For the above environment, we also needed to delete `-p docker` from [line 14 of the Makefile](https://github.com/posm/posm-build/blob/master/live/Makefile#L14).
```

Comment:
You can actually make this change to eliminate this line (since it's redundant with what's in there).

Reply:
line 58 deleted